### DISCUSSION TITLE
电脑开启HDR后，播放视频过亮

### DISCUSSION THREAD

**Discussion by Alex-wangwang** [diana7127/mpv.net-DW#32] [Q&A]
如题目所示，打开电脑的HDR功能，播放任意一个视频，视频亮度过亮，不能直视，关闭HDR功能，视频亮度回复正常。测试其他播放器，如pot，则是正常的。

**Comment by StrawWay** [comment]
我也是同样的问题，感觉饱和度很高，色彩不准。

**Reply by Elics-D** [reply]
设置，色彩管理icc-profile-auto选择no（自动使用系统当前icc文件），就行了

**Reply by chenForbear** [reply]
选no后的确不过亮了，甚至对比potplayer还稍暗一些。但是一旦暂停就会重新回到亮度瞎眼的画面。

**Reply by SIEGBAR** [reply]
> 选no后的确不过亮了，甚至对比potplayer还稍暗一些。但是一旦暂停就会重新回到亮度瞎眼的画面。

你差了一步，还要在 target-trc 中填写 pq
